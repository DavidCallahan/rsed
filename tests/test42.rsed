print "line 1\nline 2"
print "line 1\nline 2"r
