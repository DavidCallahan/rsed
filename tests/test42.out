line 1
line 2
line 1\nline 2
